Content.Shared/Damage/DamageSpecifier.cs

```diff
@@ -147,10 +147,10 @@ public static DamageSpecifier ApplyModifierSet(DamageSpecifier damageSpec, Damag
                 float newValue = value.Float();
 
                 if (modifierSet.FlatReduction.TryGetValue(key, out var reduction))
-                    newValue = Math.Max(0f, newValue - reduction); // flat reductions can't heal you
+                    newValue = Math.Max(0f, newValue - reduction*br); // flat reductions can't heal you
 
                 if (modifierSet.Coefficients.TryGetValue(key, out var coefficient))
-                    newValue *= coefficient; // coefficients can heal you, e.g. cauterizing bleeding
+                    newValue *= 1-coefficient*br; // coefficients can heal you, e.g. cauterizing bleeding
```

Comment:
This is sussy and probably will never work as intended, instead making damage heal you.